app/models/core_data_connector/name.rb

```diff
@@ -0,0 +1,9 @@
+module CoreDataConnector
+  class Name < ApplicationRecord
+    has_many :source, through: :source_titles, source: :nameable, source_type: 'Instance'
```

Comment:
Can you explain a little about what is going on here? I didn't think multiple `has_many` relationships with the same name could be valid.

Reply:
Just some dead code from a very early version of the feature - removed.